ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/forkchoice/PayloadAttributesCalculator.java

```diff
@@ -113,4 +116,16 @@ public void updateProposers(
           new SlotAndBlockRoot(spec.computeStartSlotAtEpoch(requiredEpoch), head.getRoot()));
     }
   }
+
+  public List<Map<String, Object>> getData() {
+    return proposerInfoByValidatorIndex.entrySet().stream()
+        .map(
+            proposerInfoEntry ->
+                ImmutableMap.<String, Object>builder()
+                    .put("proposerIndex", proposerInfoEntry.getKey().intValue())
+                    .put("feeRecipient", proposerInfoEntry.getValue().feeRecipient.toHexString())
+                    .put("expirySlot", proposerInfoEntry.getValue().expirySlot.intValue())
+                    .build())
```

Comment:
to Encapsulate this I would just create a schema object with the data types and return that rather than a map, then the swagger could be well defined.

Reply:
You'll want to return strings rather than ints, i'm guessing they're uint64, so the object would have 2 x uint64 and a bytes20.
The JsonProvider will generate basically 3 strings, and swagger just needs the `@Schema` annotation to define
`@Schema(type = "string", format = "uint64")`
`@Schema(type = "string", format = "byte", description = DESCRIPTION_BYTES20)`
The bytes20 description is likely not there, but can use 'DESCRIPTION_BYTES32' to find where they are :)